.github/workflows/deploy_combined_service.yml

```diff
@@ -9,6 +9,11 @@ on:
           required: true
           options:
           - dev
+        run_e2e_tests:
+          description: Run end-to-end tests?
+          type: boolean
+          default: true
+          required: true
```

Comment:
This adds a dropdown to choose if we want to run the e2e tests, right?

Reply:
Yeah believe so